routes/web.php

```diff
@@ -20,6 +20,7 @@
     Route::get('/home', 'AdminController@index');
     Route::resource('/categories', 'CategoryController', ['as' => 'admin']);
     Route::resource('/news', 'NewsController', ['as' => 'admin']);
+    Route::resource('/users', 'UserController', ['as' => 'admin']);
```

Comment:
pls use `only` or `except` to reduce the number of routes https://laravel.com/docs/5.4/controllers#resource-controllers